Formula/rpki-client.rb

```diff
@@ -0,0 +1,55 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://www.rpki-client.org/index.html"
+  url "https://ftp.openbsd.org/pub/OpenBSD/rpki-client/rpki-client-7.4.tar.gz"
+  sha256 "efd336137707ad73fa51f8480cff686c21fea7086619432069c1a02553b68d57"
+  license "ISC"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
```

Comment:
Are these still needed?

Reply:
No. with the other changes, they have been removed now. Thanks!